test/controllers/user.test.js

```diff
@@ -524,4 +525,179 @@ describe('Users', () => {
       );
     });
   });
+  describe('GET /users/:user_id/albums/:album_id/photos', () => {
+    it('Given an admin user should see its album photos', done => {
+      usersHelpers.buyAlbumAsAdmin().then(({ admin, albumPurchase, authorization }) => {
+        albumPurchasesHelpers.mockPhotosToSuccess(albumPurchase.albumId);
+        return chai
+          .request(server)
+          .get(`/users/${admin.id}/albums/${albumPurchase.albumId}/photos`)
+          .set(sessionManager.HEADER_NAME, authorization)
+          .send()
+          .then(res => {
+            res.should.have.status(200);
+            res.should.be.json;
+            res.body.should.be.instanceOf(Array);
```

Comment:
Why do you don't check elements of array ?

Reply:
Solved!